apps/payment/models.py

```diff
@@ -362,10 +375,12 @@ def _handle_status_change(self):
             """ Handle the completed payment. Remove delays, suspensions and marks """
             self.payment.handle_payment(self.user)
             self.status = status.DONE
+            self._create_fiken_sale(self.payment_price.price)
```

Comment:
Should this be done before the status is set? Just so that in case something here goes wrong, it's not marked as DONE?

Reply:
Yeah, that sounds reasonable